Assets/Scripts/GameData/Abilities/CombatAbilities.cs

```diff
@@ -20,23 +18,22 @@ public class CombatAbilities : IAbility
         public CombatAbilities(int inputID)
         {
             DatabaseConnection conn = new DatabaseConnection();
-            DatabaseReader reader = conn.QueryRowFromTableWithID("Physical_Abilities", inputID);
+            DatabaseReader reader = conn.QueryRowFromTableWithID("Abilities", inputID);
 
             ID = inputID;
             if (reader.NextRow())
             {
-                //int descriptorID = reader.GetIntFromCol("Descriptor_FK");
-                //Descriptor = new Descriptor(descriptorID);
-
-                weaponID = reader.GetIntFromCol("Weapon_FK");
-                //Weapon = new WeaponTable(weaponID);
+                Name = reader.GetStringFromCol("Name");
+                Description = reader.GetStringFromCol("Description");
+                FlavorText = reader.GetStringFromCol("Flavor_Text");
 
                 int attackModifiersID = reader.GetIntFromCol("Attack_Modifiers_FK");
                 AttackModifiers = new AttackModifiers(attackModifiersID);
 
                 Damage = reader.GetIntFromCol("Damage");
                 Accuracy = reader.GetIntFromCol("Accuracy");
                 Range = reader.GetIntFromCol("Range");
+                IsPhysical = reader.GetIntFromCol("Is_Phsycial") > 0;

```

Comment:
"Is_Physical" is misspelled inside the quotes on line 36

Reply:
Gracias